CMakeLists.txt

```
@@ -684,17 +684,30 @@ set(BOUT_HAS_ARKODE ${BOUT_USE_SUNDIALS})
 set(BOUT_HAS_CVODE ${BOUT_USE_SUNDIALS})
 set(BOUT_HAS_IDA ${BOUT_USE_SUNDIALS})
 
-option(BOUT_USE_NLS "Enable Native Language Support" ON)
+set(ON_OFF_AUTO ON OFF AUTO)
+set(BOUT_USE_NLS AUTO CACHE STRING "Enable Native Language Support")
+set_property(CACHE BOUT_USE_NLS PROPERTY STRINGS ${ON_OFF_AUTO})
+
 if (BOUT_USE_NLS)
+  set(BOUT_HAS_GETTEXT OFF)
   find_package(Gettext)
   if (GETTEXT_FOUND)
-	find_package(Intl)
-	if (Intl_FOUND)
-	  target_link_libraries(bout++
-		PUBLIC ${Intl_LIBRARIES})
-	  target_include_directories(bout++
-		PUBLIC ${Intl_INCLUDE_DIRS})
-	endif()
+    find_package(Intl)
+    if (Intl_FOUND)
+      target_link_libraries(bout++
+        PUBLIC ${Intl_LIBRARIES})
+      target_include_directories(bout++
+        PUBLIC ${Intl_INCLUDE_DIRS})
+      set(BOUT_HAS_GETTEXT ON)
+    else()
+      if (BOUT_USE_NLS == "ON")
```

```suggestion
      if (BOUT_USE_NLS)
```

This should allow all the other truthy values like `on, YES, 1`, etc